src/bundle/Resources/public/scss/_modals.scss

```diff
@@ -1,3 +1,13 @@
+.ez-modal {
+    .modal-header {
+        .ez-icon {
```

Comment:
Please, move the icon styling into the `_icons.scss` file. There we keep the icon styles. Also, you'll notice that there's already the `.ez-icon--medium` class defined which might be useful in your case.

Reply:
I tested `.ez-icon--medium` as my first option, but it is not applied in all cases, because there are other CSS selectors that are more specific.